prospector/datamodel/commit_features.py

```diff
@@ -0,0 +1,11 @@
+from dataclasses import dataclass, field
+
+from datamodel.commit import Commit
+
+from . import BaseModel
+
+
+@dataclass
+class CommitFeatures(BaseModel):
+    commit: Commit
+    reference_to_vuln_id: bool = False
```

Comment:
since it's a bool, how about renaming as `references_vuln_id`?

Reply:
Good idea, will do that.